### PR TITLE
Described PROJECT_ID in README.md

### PR DESCRIPTION
**Pull request by turbaszek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ To run the app just do:
 ```shell
 PROJECT_ID="your-project-id" make build
 ```
+where `PROJECT_ID` is the id of Google Cloud project you wnat to use.
```
Comment (janusz-chludzinski) [inline README.md:24]: wnat -> want :)

File: README.md [hunk 2]
```diff
@@ -40,6 +41,7 @@ Deploy the image using Cloud Build and deploy using Cloud Run:
 ```shell
 PROJECT_ID="your-project-id" REGION="europe-west1" make deploy
 ```
+where `PROJECT_ID` is the id of Google Cloud project you wnat to use.
```
Comment (janusz-chludzinski) [inline README.md:44]: wnat -> want :)

**Review by janusz-chludzinski — APPROVED** [review]
👌 , but please fix typo 😉

File: README.md (hunk 1 above)
Comment (turbaszek) [inline README.md:24]:
```suggestion
where `PROJECT_ID` is the id of Google Cloud project you want to use.
```

File: README.md (hunk 2 above)
Comment (turbaszek) [inline README.md:44]:
```suggestion
where `PROJECT_ID` is the id of Google Cloud project you want to use.
```

**Pull request merged**